src/App.jsx

```diff
@@ -1,50 +1,32 @@
-import logo from "/logo.png";
 import "./App.css";
-import { onChildAdded, push, ref, set } from "firebase/database";
-import { database } from "./firebase";
-import { useState, useEffect } from "react";
-
-// Save the Firebase message folder name as a constant to avoid bugs due to misspelling
-const DB_MESSAGES_KEY = "messages";
+import Composer from "./Components/Composer";
+import NewsFeed from "./Components/Newsfeed";
+import AuthForm from "./Components/AuthForm";
+import { useState } from "react";
+import { createBrowserRouter, RouterProvider } from "react-router-dom";
 
 function App() {
-  const [messages, setMessages] = useState([]);
-
-  useEffect(() => {
-    const messagesRef = ref(database, DB_MESSAGES_KEY);
-    // onChildAdded will return data for every child at the reference and every subsequent new child
-    onChildAdded(messagesRef, (data) => {
-      // Add the subsequent child to local component state, initialising a new array to trigger re-render
-      setMessages((prevState) =>
-        // Store message key so we can use it as a key in our list items when rendering messages
-        [...prevState, { key: data.key, val: data.val() }]
-      );
-    });
-  }, []);
-
-  const writeData = () => {
-    const messageListRef = ref(database, DB_MESSAGES_KEY);
-    const newMessageRef = push(messageListRef);
-    set(newMessageRef, "abc");
-  };
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
+  const [user, setUser] = useState({});
 
-  // Convert messages in state to message JSX elements to render
-  let messageListItems = messages.map((message) => (
-    <li key={message.key}>{message.val}</li>
-  ));
+  const router = createBrowserRouter([
+    {
+      path: "/",
+      element: <AuthForm setUser={setUser} setIsLoggedIn={setIsLoggedIn} />,
+    },
+    {
+      path: "/form",
+      element: <Composer setIsLoggedIn={setIsLoggedIn} setUser={setUser} />,
+    },
+  ]);
 
```

Comment:
Great implementation of the Router, makes sense to have the NewsFeed always on the page. 

I believe you can remove line 26 and then change style